### PR TITLE
Fixes `ReferenceError`s in a livesample

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Fixes multiple reference errors on page https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Traversing_an_HTML_table_with_JavaScript_and_DOM_Interfaces#javascript

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks @OnkarRuikar ! 

The alernative, which might be better but is a little more work, is to isolate the example in its own section. We could have, for example:

```
## Creating an HTML table dynamically

### Example

#### HTML

#### JavaScript

#### Result

{{EmbedLiveSample(...)}}

### Explanation

...explanation with code snippets...

```

This is arguably a better organization anyway, and lets us keep the correct language indications.

**Comment by OnkarRuikar** [comment]
I've done the restructuring. The second live sample was suffering from the same thing so I've updated that as well.

As per the original document Mozilla holds the copyright. So, do we need the section 'Original Document Information'?

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md [hunk 1]
```diff
@@ -59,13 +60,25 @@ function generate_table() {
...
+```css hidden
+table {
+  margin: 1rem auto;
+}
+
+td {
+  padding: 0.5rem;
+}
+```
+
```
Comment (wbamberg) [inline files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:78]:
```suggestion
#### Result

```

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md [hunk 2]
```diff
@@ -142,55 +155,48 @@ You can build this table and its internal child elements by using just a few DOM
...
+  
+  // set the inline style
+  secondParagraph.style.background = "red";
 }
 ```
 
-{{ EmbedLiveSample('Example_Setting_the_background_color_of_a_paragraph') }}
-
-In this example, we set the `myP` variable to the DOM object for the second `p` element inside the body:
+{{ EmbedLiveSample('Example_2') }}
```
Comment (wbamberg) [inline files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:173]:
```suggestion
#### Result

{{ EmbedLiveSample('Example_2') }}
```

**Comment by wbamberg** [comment]
> I've done the restructuring. The second live sample was suffering from the same thing so I've updated that as well.

Thanks!

> As per the original document Mozilla holds the copyright. So, do we need the section 'Original Document Information'?

I think we should remove it.

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you @OnkarRuikar !

**Pull request merged**